src/behaviors/lane_change_models.jl

```diff
@@ -24,4 +24,5 @@ end
 ####################
 
 abstract type LaneChangeModel end
```

Comment:
shouldn't `LaneChangeModel` also inherit from `<: DriverModel`? If yes, we can drop the `Base.rand` implementations which are already implemented for `DriverModel`.